web/yo/app/scripts/controllers/gene.js

```diff
@@ -305,8 +306,12 @@ angular.module('oncokbApp')
 
             function setUpdatedSignature(tempArr, reviewObj) {
                 var mostRecent = mostRecentItem(tempArr);
-                reviewObj.set('mostRecentUpdatedBy', tempArr[mostRecent].get('updatedBy'));
-                reviewObj.set('mostRecentUpdateTime', tempArr[mostRecent].get('updateTime'));
+                $scope.realtimeDocument.getModel().beginCompoundOperation();
+                var timeStamp = $scope.realtimeDocument.getModel().create('TimeStamp');
+                timeStamp.value.setText(tempArr[mostRecent].get('updateTime').toString());
```

Comment:
The reason it was converted to string is to match its type. Because the value in TImeStamp is string type.